client/src/test/java/org/spine3/base/CommandsShould.java

```diff
@@ -81,10 +84,13 @@ public void have_private_ctor() {
 
     @Test
     public void pass_null_tolerance_test() {
-        NullToleranceTest.newBuilder()
-                         .setClass(Commands.class)
-                         .build()
-                         .check();
+        final boolean passed = NullToleranceTest.newBuilder()
+                                                .setClass(Commands.class)
+                                                .addDefaultValue(DEFAULT_FILE_DESCRIPTOR)
+                                                .addDefaultValue(Timestamps.getCurrentTime()) // to fulfil custom validation
+                                                .build()
```

Comment:
What are scenarios for calling `build()`, but calling `check()` never or later?

If `check()` follows `build()` in the majority of cases we can think of, I would eliminate the `build()` to prevent errors.